mp/src/game/client/neo/c_neo_player.cpp

```diff
@@ -111,6 +111,12 @@ class NeoLoadoutMenu_Cb : public ICommandCallback
 public:
 	virtual void CommandCallback(const CCommand& command)
 	{
+		auto team = GetLocalPlayerTeam();
+
+		if (team == TEAM_UNASSIGNED || team == TEAM_SPECTATOR) {
+			return;
+		}
```

Comment:
Nit: from the shareddefs header, using `if (GetLocalPlayerTeam() < FIRST_GAME_TEAM)` would be a way to express this more tersely.